ci-operator/config/openshift/configuration-anomaly-detection/configuration-anomaly-detection-master.yaml

```diff
@@ -0,0 +1,39 @@
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  secret:
+    mount_path: /tmp/secret
+    name: configuration-anomaly-detection-codecov-token
+- as: publish-coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  postsubmit: true
+  secret:
+    mount_path: /tmp/secret
+    name: configuration-anomaly-detection-codecov-token
```

Comment:
do we need two actions?

Reply:
According to the SOP we need both actions. The first is for calculating the coverage the second is for publishing it (I Assume). I might be wrong. I don't understand the CI pipeline configuration TBH.

Reply:
it's ok, I guess I would add the link to the SOP with this hard requirement.
later I will ask the release team if this is really required

Reply:
TBH, I think the whole process is a little bit over the top and is from a time where GitHub had no native CI system.
We could just use GitHub Actions... the only thing that's missing is an approval from the Openshift Org admin team.